SPECS/zlib/zlib.spec

```diff
@@ -1,13 +1,14 @@
 Summary:        Compression and decompression routines
 Name:           zlib
 Version:        1.2.12
-Release:        1%{?dist}
+Release:        2%{?dist}
 URL:            http://www.zlib.net/
 License:        zlib
 Group:          Applications/System
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Source0:        http://www.zlib.net/%{name}-%{version}.tar.xz
+Patch0:         CVE-2022-37434.patch
 %define sha1    zlib=e1cb0d5c92da8e9a8c2635dfa249c341dfd00322
```

Comment:
this can be removed if you want

Reply:
Thanks, but I've opted to not do any cosmetic fixes given that we've fixed this in 2.0 and beyond